core/Plugin/Report.php

```diff
@@ -341,21 +348,69 @@ public function render()
         return $rendered;
     }
 
+    protected function createWidget()
+    {
```

Comment:
Don't care about code in this class for now. This will be completely changed in #7822